### ISSUE TITLE
React Datepicker showing white background in theme dark mode.

### ISSUE THREAD

**Issue by jentzheng** [payloadcms/payload#6894]
### Describe the Bug

In theme dark mode, react-datepicker .react-datepicker__time stylesheet, it still showing the white backgound.

[css line of code from React-DatePicker
](https://github.com/Hacker0x01/react-datepicker/blob/main/src/stylesheets/datepicker.scss#L284)

![Screenshot 2024-06-22 at 23 53 22](https://github.com/payloadcms/payload/assets/665738/18780692-3db1-4536-aa24-01e5814938b8)

### To Reproduce

- switch to dark mode
- add a date field in a collection
```
fields: [
            {
              name: 'startAt',
              type: 'date',
              required: true,
              admin: {
                date: {
                  pickerAppearance: 'dayAndTime',
                },
                width: '50%',
              },
            },
            {
              name: 'endAt',
              type: 'date',
              required: true,
              admin: {
                date: {
                  pickerAppearance: 'dayAndTime',
                },
                width: '50%',
              },
            },
          ],
```

### Payload Version

3.0.0-beta.53

**Comment by paulpopus** [comment]
Hey @jentzheng I can't reproduce this having tried a few options on the latest beta. Is there anything more specific about your setup to reproduce this?

I will close this issue for now but happy to re-open once there's a reproduction

**Comment by jentzheng** [comment]
@paulpopus I can confirmed this issue has been resolved in `3.0.0-beta.56`, thanks for the feedback.
